orders/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+import uuid
 
-# Create your views here.
+from datetime         import datetime
```

Comment:
datetime은 파이썬 내장 모듈입니다. uuid랑 같이 라인에 있게 해주세요.
```suggestion
import uuid
from datetime         import datetime

from django.views import View
```